godot-core/src/builtin/vectors/vector2i.rs

```diff
@@ -56,6 +56,31 @@ impl Vector2i {
         Self { x, y }
     }
 
+    ///Returns the aspect ratio of this vector, the ratio of x to y.
```

Comment:
I would keep docs as concise as possible and without repetition. `x / y` is probably slightly easier to understand 🙂 
```suggestion
    /// Aspect ratio: x / y, as a `real` value.
```

Reply:
Agreed, I just copied that from Godot docs😅